README.md

```diff
@@ -62,8 +62,6 @@ version: '2'
 services:
   wordpress:
     image: visiblevc/wordpress:latest
-    links:
```

Comment:
Version 2 automatically links all services on the same network. Took it out because it adds complexity to the compose file with no real benefit.